lib/rspec/mocks/argument_list_matcher.rb

```diff
@@ -46,6 +46,7 @@ def initialize(*expected_args)
         @expected_args = expected_args
         ensure_expected_args_valid!
       end
+      ruby2_keywords :initialize if respond_to?(:ruby2_keywords, true)
 
       # @api public
       # @param [Array] args
```

Comment:
I guess the build fails because the parameter was renamed in the method signature, but not here in the docs.